<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pcdpickeddlg</class>
 <widget class="QDockWidget" name="pcdpickeddlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="alignedPointsTableWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>80</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <row>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string notr="true">4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">Y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">Z</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Error</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="niheToolButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>拟合</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="lisanhuaToolButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>重置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>生成点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="undoToolButton">
        <property name="toolTip">
         <string>Close dialog (list will be lost)</string>
        </property>
        <property name="statusTip">
         <string>Close dialog (list will be lost)</string>
        </property>
        <property name="text">
         <string>撤回</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/CC/images/smallCancel.png</normaloff>:/CC/images/smallCancel.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="reduToolButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="statusTip">
         <string>Convert list to new cloud (and close dialog)</string>
        </property>
        <property name="whatsThis">
         <string>Convert list to new cloud (and close dialog)</string>
        </property>
        <property name="text">
         <string>离散化</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/CC/images/smallValidate.png</normaloff>:/CC/images/smallValidate.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
